### DISCUSSION TITLE
How to resplit the waymo dataset?

### DISCUSSION THREAD

**Discussion by ChulanZhang** [open-mmlab/mmdetection3d#2644] [Q&A]
Hi there,
We are playing with the Waymo dataset and want to use partial data for our training task.
We saw that the MMdet3D repo already provides a split here: https://drive.google.com/drive/folders/18BVuF_RYJF0NjZpt8SnfzANiakoRMf0o
However, we want to learn how this split is done based on the video scenes. We prefer to resplit the dataset based on scenes rather than sampling. Could you help provide some clues?

Thanks,

**Comment by Xiangxu-0103** [comment]
Waymo dataset has its official training/validation/test split.

**Reply by ChulanZhang** [reply]
We understand that Waymo has its official dataset split. Our question is how those Kitti format txt files are generated. We want to generate new txt files based on our own split.

**Reply by Xiangxu-0103** [reply]
We have provided a script here: https://github.com/open-mmlab/mmdetection3d/blob/main/tools/dataset_converters/waymo_converter.py#L608
